### PR TITLE
New blog post proposal highlighting recently release SQL commands

### PR DESCRIPTION
**Pull request by brijos**
Signed-off-by: Joshua Bright <jdbright@amazon.com>

### Description
[Proposal for blog post]
 
### Issues Resolved
[[project-website/issues/986](https://github.com/opensearch-project/project-website/pull/988)]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2022-09-12-sql-search-relevance-selective-aggregation-and-window-functions-in-OpenSearch.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title:  "SQL search relevance, selective aggregation, and window functions in OpenSearch"
+authors:
+- dai-chen
```
Comment (krisfreedain) [inline _posts/2022-09-12-sql-search-relevance-selective-aggregation-and-window-functions-in-OpenSearch.md:5]: short name looks like it should be "daichen"

**Review by krisfreedain — CHANGES_REQUESTED** [review]
tiny change required

File: _posts/2022-09-12-sql-search-relevance-selective-aggregation-and-window-functions-in-OpenSearch.md (hunk 1 above)
Comment (brijos) [reply _posts/2022-09-12-sql-search-relevance-selective-aggregation-and-window-functions-in-OpenSearch.md]: Adjusted

**Review by krisfreedain — APPROVED** [review]
Looks good

**Pull request merged**